preparing-to-code/environment-setup/README.md

```diff
@@ -16,23 +16,23 @@ _JumpStart: Lesson 7_
 
 ## Notes
 
-### Interactive Ruby
+### Python REPL
 
-Interactive Ruby (`irb`) is a computer program that allows the user to enter Ruby code and get results of that code immediately. It is a feature known as a REPL(read, evaluate, print loop). These kinds of programs are great for trying out new commands, practicing syntax, and evaluating solutions to small problems. It's one of the two primary ways to use Ruby to interact with a computer. The other is to put Ruby code into a file and then ask the computer to read and execute the code in the file. We will do lots of both.
+Python provides an interactive computer program that allows the user to enter Python code and get results of that code immediately. It is a feature known as a REPL (read, evaluate, print loop). These kinds of programs are great for trying out new commands, practicing syntax, and evaluating solutions to small problems. It's one of the two primary ways to use Python to interact with a computer. The other is to put Python code into a file and then ask the computer to read and execute the code in the file. We will do lots of both.
```

Comment:
Should we include a link here to repl.it(https://repl.it/) or should that go in the hello world section of learning to code?

Reply:
I'll revise this section today to just talk about [repl.it](https://repl.it) and that way we don't have to deal with installing Python or setting up VS code at all.  

We take that barrier to entry away and add functions.